pkg/services/ssosettings/api/api.go

```diff
@@ -114,6 +115,10 @@ func (api *Api) getAuthorizedList(ctx context.Context, identity identity.Request
 			continue
 		}
 
+		if provider.Provider == social.LDAPProviderName {
+			provider.Settings["type"] = "LDAP"
+		}
+
```

Comment:
I know this is a small change, but I prefer to do this check on the frontend side in AuthProvidersListPage.tsx#101 instead of modifying the API. What do you think?

Reply:
I'm not a huge fan of this. The type is a backend responsibility, but I don't have a strong stance.

Reply:
That's true, but then we will have to provide a `type` for all the other providers, not just ldap. And I don't think we have an easy way of doing so.